ingest/ingest_pipeline.py

```diff
@@ -116,10 +120,14 @@ def __init__(
         self.study_file_id = study_file_id
         self.matrix_file = matrix_file
         self.matrix_file_type = matrix_file_type
+        self.extra_log_params = {'study_id': self.study_id, 'duration': None}
         if os.environ.get('DATABASE_HOST') is not None:
             # Needed to run tests in CircleCI.  TODO: add mock, remove this
```

Comment:
Closing the loop on [earlier feedback](https://github.com/broadinstitute/scp-ingest-pipeline/pull/108/files#r447150956):

```suggestion
            # Needed to run tests in CircleCI.
            # TODO (SCP-2000): Integrate MongoDB emulator to test_ingest.py, then remove this
```

Reply:
?